.github/workflows/ci.yml

```diff
@@ -6,23 +6,22 @@ on:
     branches: [main]
 
 env:
-  MIN_COVERAGE: 0.3
+  MIN_COVERAGE: 0.24
 
 permissions:
   checks: write
   pull-requests: write
 
 jobs:
   test:
-    runs-on: [self-hosted, tartelet]
+    runs-on: self-hosted
     steps:
       - uses: actions/checkout@v2
       - name: Run tests on iOS Simulator
         shell: bash
         run: |
-          sudo xcode-select --switch /Applications/Xcode_14.3.app
           xcodebuild -scheme BuiltinRegistryGenerator -destination "platform=macOS"
-          xcodebuild test -scheme LiveViewNative-Package -sdk iphonesimulator16.4 -destination "OS=16.4,name=iPhone 14 Pro" -enableCodeCoverage YES -resultBundlePath TestResults
+          xcodebuild test -scheme LiveViewNative-Package -destination "platform=iOS Simulator,OS=17.0.1,name=iPhone 15 Pro" -enableCodeCoverage YES -resultBundlePath TestResults -skipMacroValidation -skipPackagePluginValidation
```

Comment:
It's unclear if you need the `OS=17.0.1` here. In the [uniffi branch I'm working on in liveview-native-core, I've found it wasn't needed](https://github.com/liveview-native/liveview-native-core/blob/1e7849540783cf8a9ae749aabe165c924feed853/Makefile.toml#L377-L386). If you don't specify the OS version, it'll use the default simulator runtime/sdk that's installed. So, when it's upgraded, this CI won't need to be updated.

It was also my experience that it doesn't exit with a non-zero status field when it was slightly wrong. :/